### PR TITLE
OCDB pull request review

### PR DESCRIPTION
**Pull request by timbarat**

### CODE REVIEW

File: components/wurth/691412320002M.stanza [hunk 1]
```diff
@@ -19,19 +19,17 @@ defpackage ocdb/wurth/691412320002M:
...
 
 pcb-package header-package :
-  val x0 = 2.5
-  val y0 = 3.7
-  pad p[1] : {pth-pad(1.0)} at loc((x0 + 1.905), (y0 + 2.5))
-  pad p[2] : {pth-pad(1.0)} at loc((x0 - 1.905), (y0 - 2.5))
-  layer(Courtyard(Top)) = Rectangle(9.12, 12.0, loc(x0, y0))
+  val offset = loc(0.595, 1.2)
+  pad p[1] : {pth-pad(1.0)} at loc(1.905, 2.5) * offset
+  pad p[2] : {pth-pad(1.0)} at loc((- 1.905), (- 2.5)) * offset
```
Comment (underpickled) [inline components/wurth/691412320002M.stanza:24]: Not sure I understand how this works.

File: components/wurth/691412320002M.stanza [hunk 2]
```diff
@@ -19,19 +19,17 @@ defpackage ocdb/wurth/691412320002M:
...
+  layer(Courtyard(Top)) = Rectangle(9.12, 12.0)
   ref-label()
 
 public unique pcb-component component :
-  manufacturer = "Wurth Elektronik"
-  mpn = "691412320002M"
+  manufacturer = " Wurth Elektronik"
+  mpn = "Wurth Elektronik"
```
Comment (underpickled) [inline components/wurth/691412320002M.stanza:30]: Why is the MPN being dropped for the Manufacturer name?

**Review by underpickled — APPROVED** [review]
Have a couple of questions, but looks good to me!

File: components/wurth/691412320002M.stanza (hunk 2 above)
Comment (timbarat) [reply components/wurth/691412320002M.stanza:30]: Hmm, we caught that, and I pushed a last minute commit to revert it. But I guess it didn't come through to your pull request. Sorry about that.

File: components/wurth/691412320002M.stanza (hunk 1 above)
Comment (timbarat) [reply components/wurth/691412320002M.stanza:24]: The '*' operator applies a transformation. So loc(1.905, 2.5) * loc(0.595, 1.2) results in loc(1.905 + 0.595, 2.5 + 1.2).

Comment (underpickled) [reply components/wurth/691412320002M.stanza:24]: Ok, but how were 0.595 and 1.2 derived from the original 2.5 and 3.7?

Comment (timbarat) [reply components/wurth/691412320002M.stanza:24]: The datasheet is super vague, but if you center the pins first and then offset them, I got 0.595 as the horizontal offset of the pins and 1.2 as the vertical offset of the pins.

**Review by underpickled — APPROVED** [review]

**Pull request merged**